service/storage_service.cc

```diff
@@ -6287,6 +6287,13 @@ future<std::unordered_map<sstring, sstring>> storage_service::add_repair_tablet_
         }
     }
 
+    if (!await_completion) {
+        auto duration = std::chrono::duration<float>(std::chrono::steady_clock::now() - start);
```

Comment:
What is this `duration`? The time it took to issue the repair command? Why do you think this is interesting to log?

Reply:
It might take time to issue the command through raft state machine. We retry if _group0->client().add_entry() fails.  It is not that interesting but it doesn't hurt either. Also, it makes it a bit more consistent with the one that waits for the repair to finish.